vela-templates/definitions/internal/component/config-dex-connector.cue

```diff
@@ -0,0 +1,61 @@
+"config-dex-connector": {
+	type: "component"
+	annotations: {
+		"name.config.oam.dev": "Dex Connector"
+	}
+	labels: {
+		"catalog.config.oam.dev":       "velacore-config"
+		"type.config.oam.dev":          "dex-connector"
+		"multi-cluster.config.oam.dev": "false"
+	}
+	description: "Config information to authenticate Dex connectors"
+	attributes: workload: type: "autodetects.core.oam.dev"
+}
+
+template: {
+	output: {
+		apiVersion: "v1"
+		kind:       "Secret"
+		metadata: {
+			name:      parameter.name
+			namespace: "vela-system"
```

Comment:
Should we set the namespace here? Maybe `context.namespace` is better.

Reply:
+1. Thanks.